src/app/docs/lookup/lookup-docs.component.html

```diff
@@ -82,17 +185,21 @@
         *skyThemeIf="'default'" imagePath="~/assets/img/guidelines/lookup/lookup-anatomy-default.png"
       >
         <sky-docs-anatomy-item>Input field</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Search button</sky-docs-anatomy-item>
         <sky-docs-anatomy-item>Lookup menu</sky-docs-anatomy-item>
         <sky-docs-anatomy-item>Search results</sky-docs-anatomy-item>
-        <sky-docs-anatomy-item isOptional="true">Button to create new option</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>'Show all/matches' button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">'New' button</sky-docs-anatomy-item>
       </sky-docs-anatomy>
       <sky-docs-anatomy
         *skyThemeIf="'modern'" imagePath="~/assets/img/guidelines/lookup/lookup-anatomy-modern.png"
       >
         <sky-docs-anatomy-item>Input field</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Search button</sky-docs-anatomy-item>
         <sky-docs-anatomy-item>Lookup menu</sky-docs-anatomy-item>
         <sky-docs-anatomy-item>Search results</sky-docs-anatomy-item>
-        <sky-docs-anatomy-item isOptional="true">Button to create new option</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>'Show all/matches' button</sky-docs-anatomy-item>
```

Comment:
While this button is technically optional, this is to improve backwards compatibility and the scale of breaking changes. Practically, we have no design guidelines for a scenario where this button should not be used on Lookup Field, so the design guidelines should treat it as required. It is not intended to be "optional" in any use case of Lookup Field.

Reply:
Resolving this thread because we want to leave this marked as not optional as discussed. I added an entry to the Options section to make the nature of this button more clear.